tests/test_download_topas.py

```diff
@@ -0,0 +1,36 @@
+import os
+from pathlib import Path
+from yaptide.admin.simulators import download_topas_from_s3
+
+
+def check_if_environment_variables_set() -> bool:
+    """Check if environment variables are set"""
+    result = True
+    if not 'S3_ENDPOINT' in os.environ:
+        result = False
+    if not 'S3_ACCESS_KEY' in os.environ:
+        result = False
+    if not 'S3_SECRET_KEY' in os.environ:
+        result = False
+    if not 'S3_TOPAS_BUCKET' in os.environ:
+        result = False
+    if not 'S3_TOPAS_KEY' in os.environ:
+        result = False
+    if not 'S3_GEANT_BUCKET' in os.environ:
+        result = False
+    if not 'S3_TOPAS_VERSION' in os.environ:
+        result = False
+    return result
+
+def test_if_topas_downloaded(tmpdir):
```

Comment:
that test should be skipped on Windows as we are not planning to use TOPAS windows builds